frontend/src/components/mine/Permit/MinePermitTable.js

```diff
@@ -0,0 +1,349 @@
+      );
+    },
+  },
+];
+
+const childColumns = [
+  {
+    title: "Amendment",
+    dataIndex: "amendmentNumber",
+    key: "amendmentNumber",
+    render: (text) => <div title="Amendment">{text}</div>,
+  },
+  {
+    title: "Date Issued",
+    dataIndex: "issueDate",
+    key: "issueDate",
+    render: (text) => <div title="Issue Date">{text}</div>,
+  },
+  {
+    title: "Description",
+    dataIndex: "description",
+    key: "description",
+    render: (text) => <div title="Description">{text}</div>,
+  },
+  {
+    title: "",
+    dataIndex: "amendmentEdit",
+    key: "amendmentEdit",
+    align: "right",
+    render: (text, record) => (
+      <AuthorizationWrapper inTesting>
+        <AuthorizationWrapper permission={Permission.CREATE} isMajorMine={text.major_mine_ind}>
+          <Button
+            className="permit-table-button"
+            type="ghost"
+            onClick={(event) =>
+              record.openEditAmendmentModal(
+                event,
+                text.guid,
+                text.permit_guid,
+                record.amendmentType,
+                text.description
+              )
+            }
+          >
+            <div>
+              <img className="padding-small--right icon-svg-filter" src={EDIT_OUTLINE} alt="Edit" />
+            </div>
+          </Button>
+        </AuthorizationWrapper>
+      </AuthorizationWrapper>
+    ),
+  },
+];
+
+const getPermittees = (partyRelationships, permit) =>
+  partyRelationships
+    .filter(({ related_guid }) => permit.permit_guid === related_guid)
+    .sort((order1, order2) => {
+      const date1 = Date.parse(order1.due_date) || 0;
+      const date2 = Date.parse(order2.due_date) || 0;
+      return date1 === date2 ? order1.order_no - order2.order_no : date1 - date2;
+    });
+
+const getPermitteeName = (permittees) =>
+  permittees[0] ? permittees[0].party.party_name : Strings.EMPTY_FIELD;
+
+const transformRowData = (
+  permit,
+  partyRelationships,
+  major_mine_ind,
+  openEditPermitModal,
+  openAddPermitAmendmentModal,
+  openAddAmalgamatedPermitModal,
+  permitStatusOptions
+) => {
+  const latestAmendment = permit.amendments[0];
+  const firstAmendment = permit.amendments[permit.amendments.length - 1];
+
+  const permittees = getPermittees(partyRelationships, permit);
+  const permitteeName = partyRelationships.length === 0 ? "" : getPermitteeName(permittees);
+  const hasAmalgamated = permit.amendments.find((pa) => pa.permit_amendment_type_code === "ALG");
+
+  return {
+    key: permit.permit_guid,
+    lastAmended: (latestAmendment && formatDate(latestAmendment.issue_date)) || Strings.EMPTY_FIELD,
+    permitNo: permit.permit_no || Strings.EMPTY_FIELD,
+    firstIssued: (firstAmendment && formatDate(firstAmendment.issue_date)) || Strings.EMPTY_FIELD,
+    permittee: permitteeName,
+    authorizationEndDate:
+      latestAmendment && latestAmendment.authorization_end_date
+        ? formatDate(latestAmendment.authorization_end_date)
```

Comment:
One remaining redundant null check

Reply:
Removed. Thought the conditional was still needed for the empty string but it's not.